stdlib/3/unittest/__init__.pyi

```diff
@@ -81,12 +81,14 @@ class TestCase:
                      msg: Any = ...) -> _AssertRaisesContext[_E]: ...
     @overload
     def assertRaisesRegex(self,  # type: ignore
-                          exception: Union[Type[BaseException], Tuple[Type[BaseException], ...]],
+                          expected_exception: Union[Type[BaseException], Tuple[Type[BaseException], ...]],
+                          expected_regex: Union[str, bytes],
```

Comment:
You're missing the `Pattern` option now, which was in an earlier iteration. The documentation (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex) explicitly says that a pattern object is acceptable. I think the union can also include `Pattern[bytes]` and `Pattern[str]`.

Reply:
I will make the changes when I reach home.